openwisp_firmware_upgrader/admin.py

```diff
@@ -45,8 +51,9 @@ class BaseVersionAdmin(MultitenantAdminMixin, TimeReadonlyAdminMixin, VersionAdm
 @admin.register(load_model('Category'))
 class CategoryAdmin(BaseVersionAdmin):
     list_display = ['name', 'organization', 'created', 'modified']
-    list_filter = [('organization', MultitenantOrgFilter)]
+    list_filter = [MultitenantOrgFilter]
```

Comment:
This is looking weird to me:
![Screenshot from 2022-12-10 13-12-00](https://user-images.githubusercontent.com/841044/206864465-e3185216-e4ea-4a8c-a52c-4c347c8bb027.png)

Reply:
@nemesisdesign Could you please double check this? because I'm getting the right autocomplete filters, see below:

![Screenshot from 2022-12-11 16-11-29](https://user-images.githubusercontent.com/56113566/206899151-c7217004-9546-4452-85fc-c78b56825c14.png)

![Screenshot from 2022-12-11 16-10-44](https://user-images.githubusercontent.com/56113566/206899109-c4ae96b1-2e06-40a2-855f-7d45c98da554.png)

Reply:
Thanks for your patience, something was wrong with my setup, all looks good, I am going to ask @pandafy to double check this since he implemented the base code for this, thanks! :+1: